PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJetSetup.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import division
+
+import six
+
+from RecoHI.HiJetAlgos.HiRecoPFJets_cff import akCs4PFJets
+from RecoHI.HiJetAlgos.HiGenJets_cff import ak5HiGenJets
+from RecoHI.HiJetAlgos.HiGenCleaner_cff import heavyIonCleanedGenJets
+from PhysicsTools.PatAlgos.producersHeavyIons.heavyIonJets_cff import *
+from PhysicsTools.PatAlgos.producersLayer1.jetProducer_cfi import *
+from PhysicsTools.PatAlgos.recoLayer0.jetCorrFactors_cfi import *
+from PhysicsTools.PatAlgos.mcMatchLayer0.jetFlavourId_cff import *
+from PhysicsTools.PatAlgos.mcMatchLayer0.jetMatch_cfi import *
+from PhysicsTools.PatAlgos.tools.helpers import *
+
+from RecoBTag.Configuration.RecoBTag_cff import *
+from RecoJets.JetAssociationProducers.ak5JTA_cff import *
+
+def setupHeavyIonJets(process, tag, radius, task):
+    addToProcessAndTask(
+        tag + 'Jets',
+        akCs4PFJets.clone(rParam = radius / 10),
+        process,
+        task)
+
+    modules = {
+        'JetTracksAssociatorAtVertex':
+        ak5JetTracksAssociatorAtVertex.clone(
+            jets = tag + "Jets",
+            tracks = "highPurityGeneralTracks",
+            ),
```

Comment:
the result here comes out with `coneSize = 0.5`, independent of the jet cone size.
Is it really intentional?

Reply:
It's intentional to the extent that we've used 0.5 throughout Run 2, independent of jet cone size.  It's possible this was originally an unintentional feature, but we should not change it for this legacy production.  
Run 3 is a different story, but I think the candidate-based tagging doesn't use a cone, but rather uses the constituents explicitly, so it should be a moot point.

Reply:
is there a need to support cones other than 0.4, especially in a production setup for the same patJets collection (as e.g. an alternative production version)?

Reply:
Not at the moment, but perhaps it would be better to put the cone radius in the name, just in case such a need arises?